docker/Dockerfile

```diff
@@ -42,6 +42,7 @@ ADD ./static /data/project/osmose/frontend/static
 RUN npm install && \
     npm run build
 
+ADD ./views /data/project/osmose/frontend/views
 ADD ./po /data/project/osmose/frontend/po
```

Comment:
This two ADD must be replaced by the ADD of the whole project, since the Makefile look for views but also for *.py.
The ADD for the whole project follows, it must be before the make po.

Reply:
Indeed, I should have fixed the PR.